test/lib.bash

```diff
@@ -170,6 +180,51 @@ function end_prober_test {
   wait ${PROBER_PID}
 }
 
+function run_knative_serving_operator_tests {
+  (
+  local version target serverless_rootdir exitstatus patchfile fork gitdesc
+  version="$1"
+  fork="${2:-knative}"
+  serverless_rootdir="$(pwd)"
+  make_temporary_gopath
+
+  logger.info "Checkout the code ${fork}/serving-operator @ ${version}"
+  mkdir -p "${GOPATH}/src/knative.dev"
+  target="${GOPATH}/src/knative.dev/serving-operator"
+  git clone --branch "${version}" --depth 1 \
+    "https://github.com/${fork}/serving-operator.git" \
+    "${target}"
+  pushd "${target}" || return $?
+
+  patchfile="${serverless_rootdir}/test/patches/SRVKS-241-knative-serving-operator-skip-configure.patch"
+  logger.info "Apply walkaround for SRVKS-241"
+  logger.debug "Patchfile is: ${patchfile}"
+  [ -f "${patchfile}" ] || return $?
+  patch --strip=0 < "${patchfile}" || return $?
+
+  gitdesc=$(git describe --always --tags --dirty)
+
+  exitstatus=0
+
+  logger.info "Run tests of knative/serving-operator @ ${gitdesc}"
+  env TEST_NAMESPACE='knative-serving' \
+    go test -v -tags=e2e -count=1 -timeout=30m -parallel=1 ./test/e2e \
```

Comment:
Let's replace this line with `go_test_e2e -tags=e2e -timeout=30m -parallel=1 ./test/e2e \`
That way we'll have JUnit reports generated.

Reply:
Sure.

Reply:
We can fix this in the other PR as well.